### PR TITLE
Layout dashboard and view category

### PR DESCRIPTION
**Pull request by quyensama**
## Related Tickets:
- [#45906](https://edu-redmine.sun-asterisk.vn/issues/45906)

## WHAT this PR does?
- Layout dashboard and view category

## Notes Impacted Areas:
- N/A.

## Attachments:
image

### CODE REVIEW

File: resources/views/dashboards/admin/categories/edit.blade.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                    </form>
+                </div>
+                <!-- /.card-body -->
+            </div>
+            <!-- /.card -->
+        </div>
+    </div>
+</section>
+<!-- /.content -->
+@endsection
```
Comment (NamBZ) [inline resources/views/dashboards/admin/categories/edit.blade.php]: eof

**Review by NamBZ — APPROVED** [review]

**Comment by quyensama** [comment]
ready

**Comment by quyensama** [comment]
ready

File: resources/views/dashboards/admin/categories/create.blade.php [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+                    </form>
+                </div>
+                <!-- /.card-body -->
+            </div>
+            <!-- /.card -->
+        </div>
+    </div>
+</section>
+<!-- /.content -->
+@endsection
```
Comment (ghost) [inline resources/views/dashboards/admin/categories/create.blade.php]: bổ sung dòng trắng cuối file

File: resources/views/dashboards/admin/categories/edit.blade.php (hunk 1 above)
Comment (ghost) [reply resources/views/dashboards/admin/categories/edit.blade.php]: chưa sửa nè

**Comment by quyensama** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**